module/project/Dependencies.scala

```diff
@@ -11,7 +11,7 @@ object Dependencies {
 
   //versions
 
-  val playVersion = "2.6.0"
+  val playVersion = "2.6.7"
```

Comment:
Might as well go with the latest Play, v2.6.13?

https://www.playframework.com/download#older-versions